### DISCUSSION TITLE
Is it possible to get do/end blocks autogenerated for Elixir

### DISCUSSION THREAD

**Discussion by formoxyz** [helix-editor/helix#4255] [Q&A]
Hello,

I am trying to use Helix for Elixir development.
Elixir functions are defined like so
def my_func() do
end

Is it possible to get the **end** auto generated as soon as we press enter after **do**?

**Comment by the-mikedavis** [comment]
Not currently. This would either need #4035 or it might be solved by #395 if ElixirLS sends do/end completions as snippets
